internal/service/streamconnection/resource_stream_connection_test.go

```diff
@@ -65,6 +66,10 @@ func testCaseKafkaPlaintext(t *testing.T) *resource.TestCase {
 		},
 	}
 }
+func DummySleep(*terraform.State) error {
```

Comment:
Merge blocker: Getting 500s when a stream connection changes networking to VPC and is deleted straight afterwards

Reply:
removed in 86e0e357d9ce9399394228edea56023fc88269e4